src/Tablebot/Plugins/Basic.hs

```diff
@@ -43,9 +47,17 @@ basicCommands =
       "You can make a pull request for that!",
       Simple ("you know what to do", "You know what to do")
     ),
+    ( "issue",
+      "You can submit an issue for that!",
+      Simple ("you know what you want someone else to do", "You know what you want someone else to do")
+    ),
     ( "benji",
-      ":benji_sit:",
+      "<:benji_sit:920000993721196654>",
```

Comment:
Hmm. I can see why you did this, but I'm not sure I like this...

Reply:
Happy to change in a future PR.